rdrf/rdrf/templates/registration/registration_login_details.html

```diff
@@ -1,7 +1,35 @@
 {% load i18n static %}
 
+<link rel="stylesheet" href="{% static 'vendor/bootstrap-select-1.14.0-beta3/css/bootstrap-select.min.css' %}">
+
 <script type="text/javascript" src="{% static 'js/show_password.js' %}"></script>
+<script id="id_bootstrap_selector_js" type="text/javascript" src="{% static 'vendor/bootstrap-select-1.14.0-beta3/js/bootstrap-select.min.js' %}"></script>
+
 <script>
+    const scriptTag = document.createElement('script');
+    scriptTag.type = "text/javascript";
+    scriptTag.setAttribute("async", "false");
+    scriptTag.src = `/static/vendor/bootstrap-select-1.14.0-beta3/js/i18n/defaults-${navigator.languages[0].replace('-', '_')}.min.js`;
```

Comment:
Does this script need to change based on the language selected by the user in the Language dropdown next to the Login button (top right)

![image](https://github.com/eresearchqut/trrf/assets/60163872/c207a9a3-8c1d-482d-b1fe-22ad267d3ee5)